Patch to fix a few warnings.
Fix "cast to pointer from integer of different size" warning.
==========================================================================
--- a/test/putsurface/putsurface.c
+++ b/test/putsurface/putsurface.c
@@ -241,7 +241,7 @@
         while (surface_id == VA_INVALID_SURFACE)
             surface_id = get_next_free_surface(&index);
 
-        if (verbose) printf("Thread %x Display surface 0x%p,\n", (unsigned int)win, (void *)surface_id);
+        if (verbose) printf("Thread %x Display surface 0x%p,\n", (unsigned int)win, &surface_id);
 
         upload_surface(va_dpy, surface_id, box_width, row_shift, display_field);
 
